### PR TITLE
trim whitespace from y axis and scale image

### PR DESCRIPTION
**Pull request by wkurchied**

### CODE REVIEW

File: src/_lib/modules/word-cloud/word-cloud.component.ts [hunk 1]
```diff
@@ -65,16 +65,31 @@ export class WordCloudComponent<T extends IWord> implements OnChanges {
...
         this._positionedWords.forEach(pw => {
             const point: IPoint = { x: pw.canvasX, y: pw.canvasY };
             this._setFontDetails(exportCtx, pw.fontSize);
             this._drawWord(pw, point, exportCtx);
         });
 
-        return exportCanvas.toDataURL('image/png');
+        const lowestPointOnTheYAxis = Math.min(...this._positionedWords.map(pw => pw.canvasY - pw.height / 2));
+        const highestPointOnTheYAxis = Math.max(...this._positionedWords.map(pw => pw.canvasY + pw.height / 2));
+        const hightOfWordCloudContent = (highestPointOnTheYAxis - lowestPointOnTheYAxis) * ratio;
```
Comment (garethdn) [inline src/_lib/modules/word-cloud/word-cloud.component.ts:82]: typo, should be `heightOfWordCloudContent`

File: src/_lib/modules/word-cloud/word-cloud.component.ts [hunk 2]
```diff
@@ -65,16 +65,31 @@ export class WordCloudComponent<T extends IWord> implements OnChanges {
...
+        exportCtx.scale(ratio, ratio);
 
         this._positionedWords.forEach(pw => {
             const point: IPoint = { x: pw.canvasX, y: pw.canvasY };
             this._setFontDetails(exportCtx, pw.fontSize);
             this._drawWord(pw, point, exportCtx);
         });
 
-        return exportCanvas.toDataURL('image/png');
+        const lowestPointOnTheYAxis = Math.min(...this._positionedWords.map(pw => pw.canvasY - pw.height / 2));
```
Comment (garethdn) [inline src/_lib/modules/word-cloud/word-cloud.component.ts:80]: This could generally do with some docs to explain what we're doing and why

**Review by garethdn — CHANGES_REQUESTED** [review]

File: src/_lib/modules/word-cloud/word-cloud.component.ts (hunk 2 above)
Comment (wkurchied) [reply src/_lib/modules/word-cloud/word-cloud.component.ts:84]: @garethdn  Let me know if i need to add more info and ill do that

Comment (garethdn) [reply src/_lib/modules/word-cloud/word-cloud.component.ts:84]: Looks good. Couple of typos: heighest -> highest, y-axies -> y-axis

**Review by garethdn — APPROVED** [review]

File: src/_lib/modules/word-cloud/word-cloud.component.ts (hunk 2 above)
Comment (wkurchied) [reply src/_lib/modules/word-cloud/word-cloud.component.ts:84]: opps :D ill update that thanks

**Pull request merged**